vendor/src/github.com/microsoft/hcsshim/createprocess.go

```diff
@@ -12,30 +14,71 @@ import (
 // processParameters is use to both the input of CreateProcessInComputeSystem
 // and to convert the parameters to JSON for passing onto the HCS
 type CreateProcessParams struct {
-	ApplicationName                   string
-	CommandLine                       string
-	WorkingDirectory                  string
-	StdInPipe, StdOutPipe, StdErrPipe string
```

Comment:
Sure, this is something we can consider on the next round of changes to HCSShim. (It's in flux right now with the incoming graph/graphdriver changes).